src/api/todos.ts

```diff
@@ -6,3 +6,15 @@
 };
 
 // Add more methods here
+
+export const postTodo = (userId: number, data: any) => {
+  return client.post(`/todos?userId=${userId}`, data);
+};
+
+export const patchTodo = (todoId: number, data: any) => {
```

Comment:
Consider adding a type for the 'data' parameter instead of using 'any' to improve type safety.